src/index.ts

```diff
@@ -32,56 +33,70 @@ const resolveHeaders = (headers: Dom.RequestInit['headers']): Record<string, str
 /**
  * Fetch data using POST method
  */
-const post = async <V = Variables>(
-  url: string,
-  query: string,
-  fetch: any,
-  options: Dom.RequestInit,
-  variables?: V,
-  headers?: HeadersInit,
-  requestHeaders?: Dom.RequestInit['headers'],
-) => {
-  const body = createRequestBody(query, variables)
+const post = async <V = Variables>({
```

Comment:
Changing the signature like this is a breaking change.

Add an overload instead for now.

Reply:
Hmm but isn't this method private only? It is not exported, so I thought it would be easier to change it that way. 
I can also add it as optional last parameter, though, if you prefer!

Reply:
Oh if its not part of the API then disregard this comment.